src/extensions/Wyam.Html/ValidateLinks.cs

```diff
@@ -29,6 +34,12 @@ namespace Wyam.Html
     /// <category>Input/Output</category>
     public class ValidateLinks : IModule
     {
+        private const int MaxAbsoluteLinkRetry = 5;
+        private const HttpStatusCode TooManyRequests = (HttpStatusCode)429;
```

Comment:
HttpStatusCode doesn't have a member for 429 explicitly. Just a helper since I use it in a few spots.